embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs

```diff
@@ -28,57 +28,33 @@ public class NUIFlutterView : ImageView
         /// </summary>
         private FlutterDesktopView View { get; set; } = new FlutterDesktopView();
 
-        /// <summary>
-        /// The initial width of the view. Defaults to the parent width if the value is zero.
-        /// </summary>
-        private int InitialWidth { get; set; } = 0;
-
-        /// <summary>
-        /// The initial height of the view. Defaults to the parent height if the value is zero.
-        /// </summary>
-        private int InitialHeight { get; set; } = 0;
-
         /// <summary>
         /// Whether the view is running.
         /// </summary>
         public bool IsRunning => !View.IsInvalid;
 
-#if NUI_SUPPORT
         /// <summary>
-        /// Save last TouchEvent time.
+        /// When the view last received a touch event in milliseconds.
         /// </summary>
-        private uint lastTouchEventTime = 0;
-#endif
+        private uint _lastTouchEventTime = 0;
 
         /// <summary>
         /// The current width of the view.
         /// </summary>
-        public int Width
-        {
-            get
-            {
-                Debug.Assert(IsRunning);
-
-                return base.Size2D.Width;
-            }
-        }
+        public int Width => Size2D.Width;
```

Comment:
This part should be reworked. `Size2D` and `Position2D` might be null or contain invalid values if not explicitly set.

Note that use of object initializer lists for non-data types is generally discouraged: https://google.github.io/styleguide/csharp-style.html#object-initializer-syntax